rank_llm/run_rank_llm.py

```diff
@@ -110,7 +112,7 @@ def main(args):
 
 
 """ sample run:
-python rank_vicuna.py --model_path=checkpoints/vicuna/vicuna-7b-checkpoint-800 --dataset=dl19 --retrieval_method=bm25
+python rank_vicuna.py --model_path=../checkpoints/vicuna/vicuna-7b-checkpoint-800 --dataset=dl19 --retrieval_method=bm25
```

Comment:
Might be better to call rank_vicuna.py from the main repo folder instead of doing this. So rank_llm/rank_vicuna.py  and remove the `../`